sources/packages/web/src/views/institution/Login.vue

```diff
@@ -107,8 +95,3 @@ export default {
   },
 };
 </script>
-<style scoped>
```

Comment:
This was removed but the class name is still consumed in line 23. Is it right?

Reply:
Thanks @andrewsignori-aot! I will remove the class. Looks like no additional styles are attached, so the class is pointless now.